airflow-core/src/airflow/ui/src/i18n/locales/en/browse.json

```diff
@@ -0,0 +1,29 @@
+{
+  "auditLog":{
+    "actions": {
+      "collapseAllExtra": "Collapse all extra json",
+      "expandAllExtra": "Expand all extra json"
+    },
+    "columns":{
+      "event": "Event",
+      "extra": "Extra",
+      "tryNumber": "Try Number",
+      "user": "User",
+      "when": "When"
+    },
+    "title": "Audit Log Events"
+  },
+  "columns":{
+    "dagId": "Dag ID",
+    "mapIndex": "Map Index",
+    "runId": "Run ID",
+    "taskId": "Task ID"
```

Comment:
These all exist as common keys I believe. Let's just reuse those instead of declaring them again.